tests/testAutopas/tests/selectors/AutoTunerTest.cpp

```diff
@@ -76,8 +78,9 @@ TEST_F(AutoTunerTest, testWillRebuildDDL) {
   configs.emplace(autopas::ContainerOption::linkedCells, autopas::TraversalOption::c08, autopas::DataLayoutOption::aos,
                   autopas::Newton3Option::disabled);
 
-  autopas::AutoTuner<Particle, FPCell> autoTuner({0, 0, 0}, {10, 10, 10}, 1, 1, 0, 100, configs,
-                                                 autopas::SelectorStrategy::fastestAbs, 1000, 2);
+  autopas::FullSearch tuningStrategy(configs);
```

Comment:
this produces double free errors, as it will also be deleted in the autoTuner class, so make it shared_ptr!